app/sqaws.py

```diff
@@ -94,6 +140,25 @@ def list_ec2_instances(pricing: PricingData):
     return instances
 
 
+# Get a list of model classes representing important properties of EBS volumes (keep separate?)
```

Comment:
Should the methods `list_ec2_instances` and `list_ebs_volumes` combined? That will resolve `keep separate?` comment too.

Reply:
Just fixed that comment - I think it's easier to keep them both separate, I just missed that comment addition when I was going back through and removing the comments I wrote to myself